### PR TITLE
update tutorial with correct cell output

### PR DESCRIPTION
**Pull request by kgalliher**
Update the parcel fabric "merge" tutorial to get rid of the `unable to resume service session` error. The hosted feature service uses Postgres which was hitting a case issue creating a `session_id`.

-----

# Checklist

Please go through each entry in the below checklist and mark an 'X' if that condition has been met. Every entry should be marked with an 'X' to be get the Pull Request approved.

- [x] All `import`s are in the first cell? 
    - [x] First block of imports are standard libraries
    - [x] Second block are 3rd party libraries
    - [x] Third block are all `arcgis` imports? Note that in some cases, for samples, it is a good idea to keep the imports next to where they are used, particularly for uncommonly used features that we want to highlight.
- [x] All `GIS` object instantiations are one of the following?
    - `gis = GIS()`
    - `gis = GIS('home')` or `gis = GIS('pro')`
    - `gis = GIS(profile="your_online_portal")`
    - `gis = GIS(profile="your_enterprise_portal")`
- [x] If this notebook requires setup or teardown, did you add the appropriate code to `./misc/setup.py` and/or `./misc/teardown.py`?
- [x] If this notebook references any portal items that need to be staged on AGOL/Python API playground, did you coordinate with a Python API team member to stage the item the correct way with the `api_data_owner` user?
- [x] If the notebook requires working with local data (such as CSV, FGDB, SHP, Raster files), upload the files as items to the [Geosaurus Online Org](geosaurus.maps.arcgis.com/) using `api_data_owner` account and change the notebook to first download and unpack the files.
- [x] Code simplified & split out across multiple cells, useful comments?
- [x] Consistent voice/tense/narrative style? Thoroughly checked for typos?
- [x] All images used like `` instead of ``? All map widgets contain a static image preview? (Call `mapview_inst.take_screenshot()` to do so)
- [x] All file paths are constructed in an OS-agnostic fashion with `os.path.join()`? (Instead of `r"\foo\bar"`, `os.path.join(os.path.sep, "foo", "bar")`, etc.)
- [x] Is your code formatted using [Jupyter Black](https://www.freecodecamp.org/news/auto-format-your-python-code-with-black/)? You can use Jupyter Black to format your code in the  notebook.
- [x] **If this notebook showcases deep learning capabilities, please go through the following checklist:**
    - [x] Are the inputs required for `Export Training Data Using Deep Learning` tool published on geosaurus org (api data owner account) and added in the notebook using `gis.content.get` function?
    - [x] Is training data zipped and published as Image Collection? Note: Whole folder is zipped with name same as the notebook name.
    - [x] Are the inputs required for model inferencing published on geosaurus org (api data owner account) and added in the notebook using `gis.content.get` function? Note: This includes providing test raster and trained model.
    - [x] Are the inferenced results displayed using a webmap widget?
- [x] **IF YOU WANT THIS SAMPLE TO BE DISPLAYED ON THE DEVELOPERS.ARCGIS.COM WEBSITE**, ping @jyaistMap so he can add it to the list for the next deploy.

### CODE REVIEW

File: guide/04-feature-data-and-analysis/part5_merge_parcels.ipynb [hunk 1]
```diff
@@ -84,16 +84,16 @@
   },
```
Comment (jyaistMap) [inline guide/04-feature-data-and-analysis/part5_merge_parcels.ipynb:84]: <p>Getting this error:</p><p> </p>

<pre class="ql-syntax" spellcheck="false">---------------------------------------------------------------------------
Exception                                 Traceback (most recent call last)
~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _hydrate(self)
  18742             try:
&gt; 18743                 self._refresh()
  18744 

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _refresh(self)
  18716                 else:
&gt; 18717                     raise e
  18718 

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _refresh(self)
  18705                 else:
&gt; 18706                     dictdata = self._con.post(
  18707                         self.url, params

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in post(self, path, params, files, **kwargs)
   1525             return resp
-&gt; 1526         return self._handle_response(
   1527             resp=resp,

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in _handle_response(self, resp, file_name, out_path, try_json, force_bytes, ignore_error_key)
   1000                 errorcode = data["error"]["code"] if "code" in data["error"] else 0
-&gt; 1001                 self._handle_json_error(data["error"], errorcode)
   1002             return data

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in _handle_json_error(self, error, errorcode)
   1023         errormessage = errormessage + "\n(Error Code: " + str(errorcode) + ")"
-&gt; 1024         raise Exception(errormessage)
   1025 

Exception: User does not have permissions to access 'washingtoncounty.mapserver'.
(Error Code: 403)

During handling of the above exception, another exception occurred:

Exception                                 Traceback (most recent call last)
/var/folders/x2/p8wq0fyj2z37bm9vgf8zbhn80000gp/T/ipykernel_94456/3310369137.py in &lt;module&gt;
----&gt; 1 parcel_fabric_flc = FeatureLayerCollection(service_urls["FeatureServer"], gis)
      2 version_management_server = parcel_fabric_flc.versions
      3 
      4 new_version_name = "merge_version_1"
      5 

~/Job/repos/geosaurus/src/arcgis/features/layer.py in __init__(self, url, gis)
   4331 
   4332         try:
-&gt; 4333             if self.properties.syncEnabled:
   4334                 self.replicas = SyncManager(self)
   4335         except AttributeError:

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in properties(self)
  18727             return self._lazy_properties
  18728         else:
&gt; 18729             self._hydrate()
  18730             return self._lazy_properties
  18731 

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _hydrate(self)
  18764                     # try as a public server
  18765                     self._lazy_token = None
&gt; 18766                     self._refresh()
  18767 
  18768                 except HTTPError as httperror:

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _refresh(self)
  18715                     dictdata = self._con.get(self.url, params)
  18716                 else:
&gt; 18717                     raise e
  18718 
  18719         self._lazy_properties = _mixins.PropertyMap(dictdata)

~/Job/repos/geosaurus/src/arcgis/gis/__init__.py in _refresh(self)
  18704                     )
  18705                 else:
&gt; 18706                     dictdata = self._con.post(
  18707                         self.url, params
  18708                     )  # , token=self._lazy_token)

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in post(self, path, params, files, **kwargs)
   1524         if return_raw_response:
   1525             return resp
-&gt; 1526         return self._handle_response(
   1527             resp=resp,
   1528             out_path=out_path,

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in _handle_response(self, resp, file_name, out_path, try_json, force_bytes, ignore_error_key)
    999                     return data
   1000                 errorcode = data["error"]["code"] if "code" in data["error"] else 0
-&gt; 1001                 self._handle_json_error(data["error"], errorcode)
   1002             return data
   1003         else:

~/Job/repos/geosaurus/src/arcgis/gis/_impl/_con/_connection.py in _handle_json_error(self, error, errorcode)
   1022 
   1023         errormessage = errormessage + "\n(Error Code: " + str(errorcode) + ")"
-&gt; 1024         raise Exception(errormessage)
   1025 
   1026     def post_multipart(

Exception: User does not have permissions to access 'washingtoncounty.mapserver'.
(Error Code: 403)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1685/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (jyaistMap) [inline guide/04-feature-data-and-analysis/part5_merge_parcels.ipynb:84]: <p>The maps are not showing up in the guide.  Can you republish this guide...but with map cells, you have to do the following</p><p> </p><ul><li>Insert a blank code cell below the map, and run map.take_screenshot().</li><li>replace the map.take_screenshot() code with the code that rendered the map, in this case </li></ul><p><em>map = gis.map("Keowns, WI")</em></p><p><em>map.draw(parcels_subset)</em></p><p><em>map.zoom_to_layer(parcels_subset)</em></p><p><em>map</em></p><ul><li><em>delete the original map cell</em></li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1685/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Comment (kgalliher) [reply guide/04-feature-data-and-analysis/part5_merge_parcels.ipynb:84]: @jyaistMap I don't have control over the enterprise. The services are wiped on some interval. This is fine, I can republish, however, now, the users are no longer able to connect to the server and create content.

**Comment by jyaistMap** [comment]
* closing in deference to #1807

**Pull request closed without merging**